infrateam_first_responder.md

```diff
@@ -51,9 +51,20 @@ It *may* be that someone in a time zone further east than Palo Alto has already
 
 It is a team task to complete these updates, but the first responder needs to make sure that all codebases needing updates have updates merged and deployed. Note that some projects may need to have PRs created by hand. It may be helpful to post in the `#dlss-infrastructure` Slack channel how many updates each developer should do, given who is working that day and how many PRs there are.
 
+##### Merge 'em
 WIP script to automatically merge all dependency update PRs is currently in its own PR in access-update-scripts: https://github.com/sul-dlss/access-update-scripts/pull/104/files -- you can switch to the branch locally and use this script.  From the comments at the top, you will need a github access token.  Instructions are here:  https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line
 
-WIP Script to automatically deploy all sul-dlss PRs is here:  https://github.com/sul-dlss-labs/sdr-deploy
+##### Deploy 'em
+Use the `sdr-deploy` script to deploy all infrastructure projects (with exceptions noted below)  using capistrano to deployed environments is here:  https://github.com/sul-dlss-labs/sdr-deploy.  
```

Comment:
would either go with:

"The `sdr-deploy` script to deploy all infrastructure projects (with exceptions noted below) using capistrano to deployed environments is here"

or

"Use the `sdr-deploy` script here to deploy all infrastructure projects (with exceptions noted below) using capistrano to deployed environments"

Reply:
also extra space between ")" and "using"

Reply:
Should we include a sentence or two about needing to set SideKiq Pro credentials for Google Books and gis-robot-suite?

Reply:
probably a good idea (though might also be good to do as a separate PR, just to get this merged)

Reply:
sidekiq pro keys should be a separate PR @jermnelson - let's get this one merged.